x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/installation_stats_collector.ts

```diff
@@ -50,7 +50,6 @@ export const getInstallationStats = async (
           deployment_mode: packagePolicy.vars?.deployment?.value as string,
           package_version: packagePolicy.package?.version as string,
           created_at: packagePolicy.created_at,
-          created_by: packagePolicy.created_by,
```

Comment:
worth to mention `created_by` was added recently (today) and wasn't released - so there's no breaking change